PriceKataCalc/Program.cs

```diff
@@ -5,8 +5,8 @@ class priceCalc
     {
         public static void Main()
         {
-            product myProduct = new("ball", 20405, 15.1567,0.20);
-            Console.WriteLine($"Product price reported as {myProduct.price} before tax and {myProduct.price+myProduct.calcTax()} after {myProduct.tax*100}% tax. ");
+            product myProduct = new("ball", 20405, 15.1567,0.20,0.15);
+            Console.WriteLine($"Product price reported as {myProduct.price} before tax and {myProduct.calcPriceAfterTax()} after {myProduct.tax*100}% tax, And with {myProduct.discount*100}% discont its {myProduct.calcPriceAfterDiscount()}  ");
```

Comment:
multiplying by 100 shouldn't be done here. I would expect the Product handles it.

Reply:
in the product both the discount and tax are fractions, the multiplication is just for the print statement